### PR TITLE
Chapter 13 (Managing Data Sources) - screenshots - updates to 3.10

### PR DESCRIPTION
**Pull request by havatv**
Goal: Update chapter 13 screenshots where necessary.
Status: All screenshots should now be according to the 3.10 GUI. I have not yet been able to check the DWG import screenshot.
Have also included a description of XYZ Tiles.

Ticket(s): #2441
Fixes: #2451, #2089, #1719, #1397, #1408, #1998 (XYZ Tiles)

- [x] Backport to LTR documentation is required

### CODE REVIEW

**Comment by havatv** [comment]
Ready for review now.

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 1]
```diff
@@ -249,13 +258,26 @@ To load a layer from a file, you can:
...
-selecting ``All files (*.*)``.
+  .. _figure_raster_add:
+
+  .. figure:: img/addrasterlayerdialog.png
+     :align: center
+
+     Add Raster Layer Dialog
+
+That will bring up a standard open file dialog (see figure_vector_open_
+and figure_raster_open_), which allows you to navigate the file system
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: Nice to have a reference to the open file dialog, but does not have to be to a screenshot. Removing the screenshot is not a problem for me.

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 2]
```diff
@@ -562,39 +605,40 @@ Working with GRASS vector data is described in Section :ref:`sec_grass`.
 Database related tools
 ----------------------
 
-.. index:: Connecting to database
+.. index:: Database - connecting
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: I wanted the index entry to show up under Database (only), but with the extra information that this has to do with connecting to a database. If you would like it to appear under "Connecting", we can add that.

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 3]
```diff
@@ -615,8 +660,9 @@ To access the connection manager, click on the :guilabel:`New` button to display
    Create a New PostGIS Connection Dialog
 
 
-The parameters required for a PostGIS connection are exposed below. For the
-other database types, see their differences at :ref:`db_requirements`.
+The parameters required for a PostGIS connection are explained below.
+For the other database types, see their differences at
+:ref:`db_requirements`.
 
 * **Name**: A name for this connection. It can be the same as *Database*.
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: Good idea. Will try to remember to fix.

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 4]
```diff
@@ -628,37 +676,51 @@ other database types, see their differences at :ref:`db_requirements`.
...
   * *Allow*: Try a non-SSL connection. If that fails, try an SSL connection;
   * *Prefer* (the default): Try an SSL connection. If that fails, try a
     non-SSL connection;
   * *Require*: Only try an SSL connection.
+  * *verify_ca*:
+  * *verify_full*:
 
-* **Username**: User name used to log in to the database.
-* **Password**: Password used with *Username* to connect to the database.
+* **Authentication**, basic.
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: Good idea.  When it is ready we can remove this.

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 5]
```diff
@@ -962,12 +1038,60 @@ the map canvas.
...
-With QGIS you can have access to different types of OGC web services (WM(T)S,
-WFS(-T), CSW ...). Thanks to QGIS Server, you can also publish these services.
-Description of these capabilities and how-to are provided in chapter
-:ref:`sec_ogc`.
+With QGIS you can get access to different types of OGC web services (WM(T)S,
+WFS(-T), WCS, CSW, ...).
+Thanks to QGIS Server, you can also publish such services.
+Chapter :ref:`sec_ogc` contains descriptions of these capabilities.
 
+.. index:: XYZ Tile services
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: No. This is not an OGC standard protocol.

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 6]
```diff
@@ -962,12 +1038,60 @@ the map canvas.
...
+The XML file for OpenStreetMap looks like this::
+
+  <!DOCTYPE connections>
+  <qgsXYZTilesConnections version="1.0">
+    <xyztiles url="https://tile.openstreetmap.org/{z}/{x}/{y}.png"
+     zmin="0" zmax="19" password="" name="OpenStreetMap" username=""
+     authcfg="" referer=""/>
+  </qgsXYZTilesConnections>
+  
+A preview is available in the :guilabel:`Preview` tab in the
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: It is the Layer properties of the context menu of the XYZ tiles connection. I will try to reformulate.

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 7]
```diff
@@ -962,12 +1038,60 @@ the map canvas.
...
+:guilabel:`Layer properties` dialog that is opened with
+:guilabel:`Properties...` in the context menu.
```
Comment (havatv) [inline source/docs/user_manual/managing_data_source/opening_data.rst]:
```suggestion
:guilabel:`Layer Properties...` in the context menu.
```

File: source/docs/user_manual/managing_data_source/supported_data.rst [hunk 8]
```diff
@@ -156,36 +154,35 @@ the :file:`.prj`.
...
 table (see :ref:`browser_panel` or :ref:`vector_loading_csv`).
-But first check that the file meets the following requirements:
+First check that the file meets the following requirements:
 
 #. The file must have a delimited header row of field names.
    This must be the first line of the data (ideally the first row in
    the text file).
-#. If geometry should be enabled, the header row must contain field(s) 
-   with geometry definition. These field(s) can have any name.
+#. If geometry should be enabled, the file row must contain field(s)
```
Comment (havatv) [inline source/docs/user_manual/managing_data_source/supported_data.rst]:
```suggestion
#. If geometry should be enabled, the file must contain field(s) 
```

[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/supported_data.rst]: I will remove "row".

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 9]
```diff
@@ -281,33 +311,35 @@ Figure_vector_loaded_ shows QGIS after loading the :file:`alaska.shp` file.
...
-* You can also load specific formats like ``ArcInfo Binary Coverage``,
+* You can also load specific vector formats like ``ArcInfo Binary Coverage``,
   ``UK. National Transfer Format``, as well as the raw TIGER format of the
-  ``US Census Bureau`` or ``OpenfileGDB``. To do that, you'd need to select
-  |radioButtonOn| :guilabel:`Directory` as :guilabel:`Source type`. In this case
-  a directory can be selected in the dialog after pressing :guilabel:`Browse`.
+  ``US Census Bureau`` or ``OpenfileGDB``. To do that, you select
+  |radioButtonOn| :guilabel:`Directory` as :guilabel:`Source type`.
+  In this case, a directory can be selected in the dialog after pressing 
+  :guilabel:`...` (browse).
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: We can try.

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 10]
```diff
@@ -628,37 +674,51 @@ other database types, see their differences at :ref:`db_requirements`.
...
-   with someone. Therefore, it's advisable to save your credentials in a
+   **credentials will be visible** if, for instance, you share the project file
+   with someone. Therefore, it is advisable to save your credentials in an
    *Authentication configuration* instead (:guilabel:`Configurations` tab -
    See :ref:`authentication_index` for more details) or in a service connection
    file (see :ref:`pg-service-file` for example).
 
+* **Authentication**, configurations.
+  Choose an authentication configuration.  You can add configurations using
+  the plus button.  Choices are:
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: Added to the list

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: I have tried.

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 11]
```diff
@@ -628,37 +676,51 @@ other database types, see their differences at :ref:`db_requirements`.
...
+   with someone. Therefore, it is advisable to save your credentials in an
    *Authentication configuration* instead (:guilabel:`Configurations` tab -
    See :ref:`authentication_index` for more details) or in a service connection
    file (see :ref:`pg-service-file` for example).
 
+* **Authentication**, configurations.
+  Choose an authentication configuration.  You can add configurations using
+  the plus button.  Choices are:  
+
+  * Basic authentication
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: Did you press the plus button?

**Comment by havatv** [comment]
I hope I have address all the issues. If not, please let me know.

File: source/docs/user_manual/managing_data_source/create_layers.rst [hunk 12]
```diff
@@ -235,60 +239,62 @@ figure_create_temporary_. Then:
 
 .. figure:: img/editNewTemporaryLayer.png
    :align: center
+   :width: 5cm
```
[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/managing_data_source/create_layers.rst]: I meant you shouldn't have to handle image width. Sphinx already takes care of that.

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: removed in #4823

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 9 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: looks like it works

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: > I wanted the index entry to show up under Database (only), but with the extra information that this has to do with connecting to a database.

This is what my suggestion was doing (fixed in #4823.

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 11 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: yes, it opens dialog to set master password, first.

File: source/docs/user_manual/managing_data_source/create_layers.rst (hunk 12 above)
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/create_layers.rst]: You have fixed this, so I resolve.

File: source/docs/user_manual/managing_data_source/opening_data.rst (hunk 6 above)
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/managing_data_source/opening_data.rst]: Done in #4824.

**Pull request merged**